src/ColorInput/ColorInput.js

```diff
@@ -120,8 +123,9 @@ class ColorInput extends React.Component {
         onCancel={this.cancel}
         onConfirm={this.confirm}
         onClickOutside={this.confirm}
-        preset={preset}
-        showClear={showClear}
+        colorPickerChildren={colorPickerChildren}
```

Comment:
```suggestion
        children={colorPickerChildren}
```
this would make more sense imo

Reply:
we talked about this with @mykas and agreed that `children` wouldn't be clear enough on where the elements will be added

Reply:
i'm talking about a prop of `ColorViewer`, not `ColorPicker`

Reply:
my bad, sorry 🎳